README.md

```diff
@@ -1,178 +1,136 @@
 # CloudWeGo-Kitex
+English | [中文](README_cn.md)
 
-Kitex[kaɪt'eks] 字节跳动内部的 Golang 微服务 RPC 框架，具有**高性能**、**强可扩展**的特点，在字节内部已广泛使用。如今越来越多的微服务选择使用 Golang，我们考虑将 Kitex 开源出来丰富开源社区。如果对微服务性能有要求，又希望定制扩展融入自己的治理体系，Kitex 会是一个不错的选择。
+Kitex [kaɪt'eks] is a **High-Performance** and **Strong-Extensibility** Golang RPC framework that can help developers build microservices. As more and more organizations are now choosing Golang, if you need high-performance microservices and want to do some customization, Kitex will be a good choice.
```

Comment:
that can help => that helps
As more and more organizations are now choosing Golang, if you need high-performance microservices and want to do some customization, Kitex will be a good choice. => If the performance and extensibility are the main concerns when you develop microservices, Kitex can be a good choice.